server.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const {injectDecoratorServerSide} = require("@navikt/nav-dekoratoren-moduler/ssr");
+const path = require("path");
+
+const decoratorParams = {
+    env: process.env.DEKORATOR_MILJO || "prod",
+    simple: true,
+    feedback: false,
+    chatbot: false,
+    shareScreen: false,
+};
+
+const app = express(); // create express app
+app.disable("x-powered-by");
+
+const buildPath = path.resolve(__dirname, "build");
+
+const basePath = "/sosialhjelp/soknad";
+
+app.use(basePath, express.static(buildPath, {index: false}));
+
+app.get(`${basePath}/internal/isAlive|isReady`, (req, res) => res.sendStatus(200));
+
+app.use(basePath, (req, res, next) => {
+    injectDecoratorServerSide({
+        filePath: `${buildPath}/index.html`,
+        ...decoratorParams,
+    })
+        .then((html) => {
+            res.send(html);
+        })
+        .catch((e) => {
```

Comment:
Dette vil gi en 500-feilmelding dersom dekoratøren er utilgjengelig. Er dette ønskelig for oss? Skal vi eventuelt bare rendre appen _uten_ dekoratør?

Reply:
men får man logget inn hvis ikke dekoratøren er tilgjengelig? 🤔

Reply:
Jeg tror vi kan få logget inn (siden dette håndteres i appen og ikke i dekoratøren), men ikke logget ut?

Jeg er ikke så sikker på hva pus-dekoratør har gjort tidligere her, men er sikkert lurt om vi tar en liten diskusjon på hva defaulten _bør_ være før denne går i prod. :)